native/win_install.ps1

```diff
@@ -0,0 +1,214 @@
+Param (
+    [switch]$debug = $false
+)
+
+#
+# Constants
+#
+$win_tridactyl_dir_name = ".tridactyl"
+$batch_file_name = "native_main.bat"
+$manifest_file_name = "tridactyl.json"
+$manifest_replace_str = "REPLACE_ME_WITH_SED"
+$native_messenger_binary_name = "native_main.py"
+
+$native_messenger_files_http_base = `
+    "https://raw.githubusercontent.com/cmcaine/tridactyl/master/native"
```

Comment:
We should allow for a "local" installation that copies the native messenger from the ./native directory, so that people can develop the native messenger on Windows, or at least exit the script with a warning that the stable release is installed.

Reply:
[Done!][0].

If the script is invoked as shown below, it will copy
`native_main.py` and `tridactyl.json` from the specified directory.

This is somewhat similar to `bash install.sh local`, but allows
picking any target directory, as well as `native` to read files
from.

Thanks.

```
  PS C:\Users\<user-name>\tridactyl> .\win_install.ps1 -DebugDirBase .\native
```

[0]: https://github.com/gsbabil/tridactyl/blob/f301f6549c43aeace44e7e7e2daa1b2e70ce7af5/native/win_install.ps1#L180